src/plugin/helpers.js

```diff
@@ -108,5 +108,16 @@ export const bulmaComponentList = [
 ]
 
 export function camelCaseToDash (myStr) {
-  return myStr.replace(/([a-z])([A-Z])/g, '$1-$2').toLowerCase().replace(/([a-z])([0-9])/g, '$1-$2')
+  return myStr
+    .replace(/([a-z])([A-Z])/g, '$1-$2')
+    .toLowerCase()
+    .replace(/([a-z])([0-9])/g, '$1-$2')
+}
+
+export const isBulmaAttribute = attr => !['is-', 'has-', 'fa-'].indexOf(attr)
+
+export const getOutrEl = (outrEl, reqOutrEl, defaultEl = 'div') => {
+  return (
```

Comment:
you can remove the `return` and making it even cleaner :)

Reply:
I know, but a bit harder to read - i think

```javascript
export const getOutrEl = (outrEl, reqOutrEl, defaultEl = 'div') =>
outrEl || reqOutrEl || vueBulmaDefaultRenderElement.get(name) || defaultEl
```

Reply:
I prefer sticking to what is [advised](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions).